projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -124,6 +125,7 @@ frontend_publish_ui_image:
       changes: *frontend_data_pipelines_locations
   extends: .frontend_publish_docker_image
 
+# TODO: Enable this when https://jira.eng.vmware.com/browse/TAUR-6077 is resolved
```

Comment:
can you make a github issue and link to that.

Reply:
[linked to github issue](https://github.com/vmware/versatile-data-kit/issues/2332)